app/controlplane/pkg/authz/authz.go

```diff
@@ -368,6 +368,7 @@ var ServerOperationsMap = map[string][]*Policy{
 	"/controlplane.v1.OrganizationService/ListMemberships": {PolicyOrganizationListMemberships},
 	// Groups
 	"/controlplane.v1.GroupService/List": {PolicyGroupList},
+	"/controlplane.v1.GroupService/Get":  {PolicyGroupRead},
```

Comment:
I think `PolicyGroupRead` needs to be added to Group Maintainers, and add the logic in the service `Get` to check maintainer policies, right?

Reply:
Keeping viewers outside the scope for now.